### ISSUE TITLE
With Certmanager 0.12 deployment fails

### ISSUE THREAD

**Issue by tgruenert** [Mailu/helm-charts#12]
given:
- rancher kubernetes cluster
- working traefik with certmanager 0.12

At certificate.yaml an acme-part is set.

helm fails with
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Certificate.spec): 
unknown field "acme" in io.cert-manager.v1alpha2.Certificate.spec
```

Imho an acme-part is not defined within certificates.cert-manager.io.

**Comment by ananthunair** [comment]
Hi @tgruenert ,

I also faced the same issue, seems like the format has been changed `Certificate` resource. I removed the ACME part from the manifest, and seems to be working now. The ACME is specified in the `ClusterIssuer` resource. 

Hope this helps.

Following are the manifests i used. Also have a PR on the way for this.

`Certificate.yaml`

```
# This is the definition of the required ssl certificate for the mail system
# It will be issued by cert-manager

{{ if default false .Values.certManagerPre011 }}
apiVersion: certmanager.k8s.io/v1alpha1
{{ else }}
apiVersion: cert-manager.io/v1alpha2
{{ end }}
kind: Certificate
metadata:
  name: {{ include "mailu.fullname" . }}-certificates
spec:
  secretName: {{ include "mailu.fullname" . }}-certificates
  renewBefore: 1440h # 60d
  commonName: "{{ first .Values.hostnames }}"
{{- range .Values.hostnames }}
      - "{{ . }}"
{{- end }}
  dnsNames:
{{- range .Values.hostnames }}
  - "{{ . }}"
{{- end }}
  issuerRef:
    kind: {{ .Values.certmanager.issuerType }}
    name: {{ .Values.certmanager.issuerName }}
```

`ClusterIssuer.yaml`
```
apiVersion: cert-manager.io/v1alpha2
kind: ClusterIssuer
metadata:
  name: letsencrypt-staging
spec:
  acme:
    # You must replace this email address with your own.
    # Let's Encrypt will use this to contact you about expiring
    # certificates, and issues related to your account.
    email: user@email.com
    server: https://acme-staging-v02.api.letsencrypt.org/directory
    privateKeySecretRef:
      # Secret resource used to store the account's private key.
      name: cert-issuer-account-key
    # Add a single challege solver, HTTP01 using nginx
    solvers:
    - http01:
        ingress:
          class: nginx
```

**Comment by micw** [comment]
@ananthunair Thank you for pointing this out. This constant breaking changes in certmanager are annoying. I will remove this and drop support for certmanager <0.12 in the next version

**Comment by ananthunair** [comment]
No probs @micw , if you need add anything new to the helm-chart i can help 😄

**Comment by micw** [comment]
Fixed in version 0.0.5
